packages/vite/src/node/build.ts

```diff
@@ -406,7 +425,81 @@ async function doBuild(
             ? createMoveToVendorChunkFn(config)
             : undefined,
         ...output
+      }
+    }
+
+    // resolve lib mode outputs
+    const outputs = resolveBuildOutputs(
+      options.rollupOptions?.output,
+      libOptions,
+      config.logger
+    )
+
+    // watch file changes with rollup
+    if (config.build.watch) {
+      config.logger.info(chalk.cyanBright(`\nwatching for file changes...`))
+
+      const output: OutputOptions[] = []
+      if (Array.isArray(outputs)) {
+        for (const _output of outputs) {
```

Comment:
**thought (dx):** normally an `_` means an ignored variable. This isn't the case here.

Reply:
Done